src/DripsHub.sol

```diff
@@ -638,9 +655,16 @@ abstract contract DripsHub {
     ) internal {
         if (amtPerSec == 0) endTime = _currTimestamp();
         if (userOrAccount.isAccount) {
-            emit Dripping(userOrAccount.user, userOrAccount.account, receiver, amtPerSec, endTime);
+            emit Dripping(
+                userOrAccount.user,
+                userOrAccount.account,
+                receiver,
+                DUMMY_ASSET,
+                amtPerSec,
+                endTime
+            );
         } else {
-            emit Dripping(userOrAccount.user, receiver, amtPerSec, endTime);
+            emit Dripping(userOrAccount.user, receiver, DUMMY_ASSET, amtPerSec, endTime);
```

Comment:
Do you think we can remove the `userOrAccount` struct? Maybe only emitting events on a higher layer?

Reply:
I was planning to get rid of it when we no longer allow non-account identity, until then it's still somewhat needed and probably not worth bigger refactorings.